e2e/provision/README.md

```diff
@@ -154,10 +177,13 @@ PackageRevision name to perform the `propose` and `approve` operations:
 kpt alpha rpkg clone -n default nephio-example-packages-0fbaccf6c5e75a3eff7976a523bb4f42bb0118ce --repository mgmt regional
```

Comment:
THis is no more the good value for v8:
```kpt alpha rpkg clone -n default nephio-example-packages-7895e28d847c0296a204007ed577cd2a4222d1ea --repository mgmt regional```

Reply:
I started testing that, but I'm not getting `config-management-monitoring` and `config-management-system` namespaces on the regional cluster, is that expected or a bug?